project_agile_workflow_transitions_by_task_type/README.rst

```diff
@@ -0,0 +1,36 @@
+.. image:: https://www.gnu.org/graphics/lgplv3-147x51.png
+   :target: https://www.gnu.org/licenses/lgpl-3.0.en.html
+   :alt: License: LGPL-v3
+
+===============================================
+Project Agile Workflow Transitions By Task Type
+===============================================
+
+This module integrates module ``project_workflow_transitions_by_task_type`` with module ``project_agile``.
+
+
+Credits
+=======
+
+
+Contributors
+------------
+
+* Petar Najman <petar.najman@modoolar.com>
+* Aleksandar Gajić <aleksandar.gajic@modoolar.com>
+* Miroslav Nikolić <miroslav.nikolic@modoolar.com>
+
+Maintainer
+----------
+
+.. image:: https://odoo-community.org/logo.png
+   :alt: Odoo Community Association
+   :target: https://odoo-community.org
+
+This module is maintained by the OCA.
+
+OCA, or the Odoo Community Association, is a nonprofit organization whose
+mission is to support the collaborative development of Odoo features and
+promote its widespread use.
+
+To contribute to this module, please visit http://odoo-community.org.
```

Comment:
Many remarks from here https://github.com/OCA/web/pull/881#pullrequestreview-99094143 applies in this PR (and probably most of the PR done today :smile: )